modules/ast/src/main/scala/sangria/ast/SourceMapper.scala

```diff
@@ -1,4 +1,4 @@
-package sangria.parser
+package sangria.ast
```

Comment:
Maybe we can revert this change? This would result in much less changes.

Reply:
The `SourceMapper` belongs with the `ast` library. Well, not all of it. The `render` methods don't belong there. But the identification of the source and the content of the source belong there.

But yes, this is the main change. It was intentional.

Reply:
My _belief_ is that it isn't disruptive. I doubt that users create their own `SourceMapper`s, or use the `SourceMapper`s directly. The only reason that i could see for doing so would be internationalization of the error messages, and then there are a lot more such messages in the `Violation`s.